spec/system/users/sessions/new_spec.rb

```diff
@@ -28,6 +28,9 @@
       expect(page).to_not have_link "Logout"
       expect(page).to have_text "Invalid Email or password"
       expect(page).to have_css('div[role="alert"]')
+
+      find("svg[aria-label=Close]").click
+      expect(page).to_not have_no_content "Invalid Email or password"
```

Comment:
Can we change this away from a double negative, though I'm confused why it's working. Don't we want it to be:

```
expect(page).to_not have_content "Invalid Email or password"
```
?

Reply:
@littleforest good catch. It must be `expect(page).to have_no_content "Invalid Email or password"`

https://til.magmalabs.io/posts/ea2ce20798-behavior-of-capybaras-havenocontent-when-expected-with-tonot-and-with-to

Reply:
Cool I did not know about `have_no_content`!